react-web/src/app/core/pages/home/index.tsx

```diff
@@ -1,13 +1,30 @@
-import React from 'react'
+import { useForm } from 'react-hook-form'
+
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
 
 import { HomeTemplate } from 'components/templates/home'
 
-import logo from 'app/core/resources/logo.svg'
+const Home = (): JSX.Element => {
+  const validationSchema = yup.object().shape({
+    amountReceived: yup.number().min(1).required('This field is required.'),
```

Comment:
Would it make sense to have a const for this default message (and eventually others too) to avoid repeating it?